docs/proposals/use-cascading-deletion/README.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Use Cascading Deletion in Karmada
+
+authors:
+- "@CharlesQQ"
+
+reviewers:
+- "@robot"
+- TBD
+
+approvers:
+- "@robot"
+- TBD
+
+creation-date: 2024-07-01
+
+# Use Cascading Deletion in Karmada
+
+## Summary
+<!--
+一种级联删除机制, 类似于[级联删除](https://kubernetes.io/docs/tasks/administer-cluster/use-cascading-deletion/)
+-->
+A cascading deletion mechanism, similar to [cascade deletion](https://kubernetes.io/docs/tasks/administer-cluster/use-cascading-deletion/)
+
+## Motivation
+<!--
+默认情况下, 当用户删除Karmada控制面的资源模版之后,成员集群的资源也会被删除。但是仍然在某些场景下,用户希望仅仅删除控制面的资源,而保留成员集群资源;亦或是删除资源模版动作之后立即返回
+-->
+
+By default, when the user deletes the resource template of the Karmada control plane, the resources of the member cluster will also be deleted. However,in some scenarios,users want to delete only the control plane resources and retain the member cluster resources; or they want to return immediately after deleting the resource template action.
+
+### Goals
+<!--
+- 提供只删除控制面资源模板,而成员集群资源不删除,同时清理Karmada附加给成员集群资源的labels/annotations的能力
+- 提供控制面资源模版删除之后,不阻塞等待成员集群资源删除的能力
+- 其他直接产生work对象的资源如namespace(ns-controller开启)/cronfederatedhpa/federaredhpa/federatedresourcequota,需要有和资源模板相同的能力
+- 提供karmadactl命令行,可以执行级联删除策略, 如 `karmadactl delete deployment --cascade=orphan`
+-->
+
+- Provides the ability to delete only the control plane resource templates without deleting the member cluster resources, and at the same time clean up the labels/annotations attached to the member cluster resources by Karmada.
+- Provides the ability to not block waiting for member cluster resource deletion after the control plane resource template is deleted.
```

Comment:
I am sorry, but what does it mean to not block waiting for member cluster resource deletion?